autoload/fzf/vim.vim

```diff
@@ -407,9 +407,12 @@ endfunction
 " Colors
 " ------------------------------------------------------------------
 function! fzf#vim#colors(...)
+  let colors = split(globpath(&rtp, "colors/*.vim"), "\n")
+  if has('packages')
+      let colors += split(globpath(&packpath, "pack/*/opt/*/colors/*.vim"), "\n")
```

Comment:
We use 2-space indentation in this project. Can you update this line?

Reply:
Sure.